Source/Scene/Batched3DModel3DTileContent.js

```diff
@@ -154,6 +154,9 @@ define([
      * Part of the {@link Cesium3DTileContent} interface.
      */
     Batched3DModel3DTileContent.prototype.initialize = function(arrayBuffer, byteOffset) {
+        // Destroy expired resources if they exist.
+        destroyResources(this);
```

Comment:
Throughout the content providers, this is a bit awkward.  Is it the cleanest solution?  Would it be better to call `destroyResources` as part of `unloadContentAndMakeEmpty`?